lib/src/DOtherSide.cpp

```diff
@@ -291,6 +289,8 @@ void dos_qguiapplication_download_imageByUrl(const char *url, const char *filePa
 {
 
     QNetworkAccessManager *manager = new QNetworkAccessManager();
+    manager->setAutoDeleteReplies(true);
+    manager->deleteLater();
```

Comment:
it will be deleted too quick, the request will have no chance to complete. Manager should be deleted after receiving response or error, no earlier.

Reply:
Right, will fix

Reply:
Crucial catch @micieslak :ok_hand: 

Calling `manager->deleteLater();` in the lambda below should do the trick.

Reply:
Using that I still get the crash (so it doesn't fix the problem); got a different idea though :)